gradle.properties

```diff
@@ -0,0 +1,14 @@
+org.gradle.caching=true
```

Comment:
Let's add license header and remove the additional extra line at the end.

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-observe/pull/1/commits/ed8df9f5f8ca5121b0dd0a3a15cb43f5ed96749a